src/engine.ts

```diff
@@ -331,4 +384,15 @@ export default class Engine {
     this.players[player].pass();
     (this[Command.ChooseRoundBooster] as any)(player, booster, Command.Pass);
   }
+
+  [Command.Leech](player: PlayerEnum, leech: number) {
+    const powerLeeched = this.players[player].data.chargePower(leech);
+    const victoryPoints = `-${powerLeeched}${Resource.VictoryPoint}`;
```

Comment:
I think the decrease in victory points is `Math.max(powerLeeched - 1, 0)`. You can also now use the other constructor of `Reward`: `new Reward(Math.max(powerLeeched - 1, 0), Resource.VictoryPoint)`

Reply:
yes, you are right!